models/attachment.go

```diff
@@ -54,6 +54,19 @@ func (m *Attachment) GetContentType()(*string) {
     }
     return nil
 }
+
+// GetContentBytes gets the contentBytes property value. The content of the attachment in bytes.
+func (m *Attachment) GetContentBytes() *[]byte {
+    val, err := m.GetBackingStore().Get("contentBytes")
+    if err != nil {
+        panic(err)
```

Comment:
Why are you triggering a panic over returning the error?

Reply:
Hi @calebkiage, the method in the interface doesn't have an error in return parameters, and I followed the structure of other methods so I triggered panic here.